src/sentry/features/temporary.py

```diff
@@ -399,6 +399,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:session-replay-new-event-counts", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
     # Enable Rage Click Issue Creation In Recording Consumer
     manager.add("organizations:session-replay-rage-click-issue-creation", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=False)
+    # Enable a reduced timeout when waiting for the DOM, before reporting Rage and Dead clicks
+    manager.add("organizations:session-replay-dead-click-reduced-timeout", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, default=False, api_expose=False)
```

Comment:
I realize I'm too late, but could the timeout be set in the option itself? instead of a bool value?
The key then could be `rage-click-timeout` (are we also changing the definition for Dead? if so then `dead-click-timeout`).

Reply:
The timeout applies to both rage and dead clicks. Yeah we can make an option, since the other PRs haven't merged yet